include/SugarObjects/templates/basic/include/ConfirmOptIn.php

```diff
@@ -73,11 +73,8 @@ public function pre_display()
      */
     public function display()
     {
-        global $app_list_strings, $app_strings, $mod_strings;
+        
```

Comment:
We need these values, so that third party developers can customise the template.

Reply:
they can

Reply:
These values are not used. If someone wants to use it, he'll add that line above.